src/blog/2023/06/flowforge-1-8-released.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Enhancing Node-RED with High Availability in FlowForge 1.8
+subtitle: Empowering Node-RED with increased availability and streamlined deployment pipelines
+description: FlowForge now supports High Availibility, DevOps Pipelines and more
+date: 2023-06-08 
+authors: ["marian-demme"]
+#image: /blog/2023/05/images/release-1.8.0.png
+---
+
+The FlowForge 1.8 update introduces a suite of features designed to optimize your Node-RED experience. High Availability ensures your business-critical processes stay running under heavy workloads, while DevOps Pipelines allow for smoother, staged deployments. Additionally, the integration of a new User Interface for the Device Agent significantly streamlines the configuration process.
+
+<!--more-->
+
+## High Availability
+Are you running business-critical processes in Node-RED and need to ensure they're always available and can handle increasing workloads? We heard you! We're excited to introduce High Availability (HA) in FlowForge to support your needs.
+
+Say you're managing a large influx of HTTP requests to your Node-RED instances. With our latest update, FlowForge now ensures that these requests are distributed evenly across your instances. This means your applications are less likely to experience downtime and can handle higher traffic loads​.
+
+You might be wondering about editing flows in this new HA environment. For now, we've disabled editing in the HA environment to ensure consistency across instances. But don't worry, you can still develop your flows in a single instance and then seamlessly roll them out to your HA-ready production instance. See therefore also our next new feature, DevOps Pipelines. High Availability is our first [preview feature](https://flowforge.com/handbook/product/versioning/#preview-features), and your feedback is crucial. We encourage you to try out HA in your Node-RED instances and share your experiences with us. Your feedback will help us refine this feature and make it even better.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/zS6P3RR86vE" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+## DevOps Pipelines
+
+Have you ever worried about the stability of new updates in your Node-RED production environment? Our new feature is designed to help you manage these concerns. DevOps Pipelines allow you to manage and track multiple deployments of your Node-RED instances, progressing through stages like test, development, staging, and production with ease​.
```

Comment:
We have already talked about devops pipelines. I think we need to emphasis what is new in 1.8?